.travis.yml

```diff
@@ -0,0 +1,32 @@
+---
+language: node_js
+node_js:
+  - "8"
+
+sudo: false
+dist: trusty
+
+branches:
+  only:
+    - master
+
+matrix:
+  include:
+  - name: "npm"
+    before_install:
+      - npm config set spin false
+    install:
+      - npm link
+    script:
+      - cd ../
+      - npm init ember-app test-app
+      - cd test-app
+      - "node_modules/.bin/ember -v | grep -q 'ember-cli: 3.6'"
```

Comment:
seems like this would fail as soon as 3.7 is released 🤔

Reply:
So I was thinking what I would want to do here to make sure this is working. I didn't want to just do `ember test` because that wouldn't verify that it was installing the latest ember 🤔 

I was thinking that I'm ok with this failing when 3.7 is released but I also wondered, is there a canonical "api" anywhere that shows the current latest ember version?

Reply:
https://github.com/ember-cli/ember-source-channel-url maybe? not sure